concepts/enums/about.md

```diff
@@ -1 +1,136 @@
 # Enums
+
+Enums are cairo data type used to define a set of predefined variants for the purpose of code readability and safety. In Cairo, `enum` variants can be declared to hold different data types (uint, struct, tuples, arrays, dictionaries, core default library, other enums).
+
+The code snippet below shows how enums could be defined:
+
+### Simple example of an enum with no associated value
+
+```Rust
+    #[derive(Drop)]
+    enum Direction {
+        North,
+        East,
+        South,
+        West,
+    }
+```
+
+The `Direction` enum declared above is a simple enum with four variants: `North`, `East`, `South`, and `West`. Notice that the variants are declared in PascalCase, being the recommended naming convention. Each variant represents a distinct value of the `Direction` enum type. Each variant in this particular enum example has no associated value and be instantiated using this syntax:
+
+```Rust
+    let direction = Direction::North;
+```
+
+### Simple example of an enum with associated value
```

Comment:
Convert this subheader into a simple sentence, similar to how it's done in [The Cairo Book](https://book.cairo-lang.org/ch06-01-enums.html#enum-variants-and-values).